src/components/PopupWithForm.js

```diff
@@ -26,4 +27,12 @@ export default class PopupWithForm extends Popup {
       this.close();
     });
   }
+
+  modalLoading(isLoading) {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`renderLoading` is the name from your theory

or `set...`